lib/ansible/module_utils/crypto.py

```diff
@@ -152,6 +152,16 @@ def get_fingerprint(path, passphrase=None):
     return get_fingerprint_of_bytes(publickey)
 
 
+def get_fingerprint_from_pem_cert(data):
```

Comment:
Since you're no longer using this function, please remove it.

Reply:
done